src/logic/safe/store/actions/fetchSafe.ts

```diff
@@ -53,11 +53,12 @@ export const buildSafe = async (safeAddress: string): Promise<SafeRecordProps> =
  * @note It's being used by the app when it loads for the first time and for the Safe's data polling
  *
  * @param {string} safeAddress
+ * @param {boolean} isInitialLoad
  */
 export const fetchSafe =
   (safeAddress: string, isInitialLoad = false) =>
   async (dispatch: Dispatch<any>): Promise<Action<Partial<SafeRecordProps>> | void> => {
-    const dispatchPromises: any[] = []
+    const dispatchPromises: ((dispatch: Dispatch, getState: () => AppReduxState) => Promise<void> | void)[] = []
```

Comment:
Can can extract this type from the [`Middleware` type](https://stackoverflow.com/questions/45339448/how-do-you-create-strongly-typed-redux-middleware-in-typescript-from-reduxs-typ)?